install/operator/deploy/manifests/1.7.x/syndesisoperator.1.7.x.clusterserviceversion.yaml

```diff
@@ -0,0 +1,933 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: syndesisoperator.1.7.x
+  namespace: placeholder
+  annotations:
+    capabilities: Basic Install
+    categories: "Integration & Delivery"
+    certified: "false"
+    createdAt: 2019-05-08T16:12:00Z
+    containerImage: syndesis/syndesis-operator
+    support: Red Hat Fuse Online
+    description: Manages the installation of Syndesis, a flexible and customizable open source platform that provides core integration capabilities as a service.
+    repository: https://github.com/syndesisio/syndesis/
+    alm-examples: |
+      [{
+          "apiVersion": "syndesis.io/v1alpha1",
+          "kind": "Syndesis",
+          "metadata": {
+          	"name": "app"
+          },
+          "spec": {
+          	"integration": {
+          		"limit": 0
+          	}
+          }
+      }]
+spec:
+  displayName: Syndesis Operator
+  description: |
+    ### Syndesis operator
+    Syndesis is a flexible and customizable, open source platform that provides core integration capabilities as a service.
+
+    This operator installs as well as configures all syndesis components.
+
+    ### How to install
+    When the operator is installed (you have created a subscription and the operator is running in the selected namespace) create a new CR of Kind Syndesis (click the Create New button). The CR spec contains all defaults (see below).
+
+    ### CR Defaults
+    The CR definition is pretty simple and an empy CR will trigger a base installation
+  keywords: ['camel', 'integration', 'syndesis', 'fuse', 'online']
+  version: 1.7.x
+  maturity: alpha
+  maintainers:
+  - name: Red Hat Fuse Online
+    email: fuse-online@redhat.com
```

Comment:
```suggestion
    email: syndesis@googlegroups.com
```

Reply:
not sure about what we want to reference here, @heiko-braun ?